crates/chain/src/chain_data.rs

```diff
@@ -131,6 +140,9 @@ impl From<(&u32, &BlockHash)> for BlockId {
 }
 
 /// An [`Anchor`] implementation that also records the exact confirmation height of the transaction.
+/// When a transaction is anchored in [`Self::anchor_block`], it does not mean
+/// it is confirmed in that block. It just means [`Self::anchor_block`] is in the best chain and
+/// therefore our transaction is also in the best chain.
```

Comment:
https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html#documenting-components

Can we stick to this structure when writing documentation?

We want the first section to be a short sentence, and have a more detailed description in a separate section.

Please also fix this for `ConfirmationTimeAnchor`.

Reply:
```suggestion
///
/// Refer to [`Anchor`] for more.
```

Instead of including this explanation for every struct that implements `Anchor`, I think it's better to move this description to `Anchor`'s documentation. We can just have "Refer to Anchor for more" at the end here.